Configuration/DataProcessing/python/Impl/hcalnzsEra_Run2_2018_pp_on_AA.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+"""
+_hcalnzsEra_Run2_2018_pp_on_AA_
+
+Scenario supporting 2018 heavyIon collisions
+
+"""
+
+import os
+import sys
+
+from Configuration.DataProcessing.Impl.hcalnzs import hcalnzs
+from Configuration.Eras.Era_Run2_2018_pp_on_AA_cff import Run2_2018_pp_on_AA
+
+class hcalnzsEra_Run2_2018_highBetaStar(hcalnzs):
+    def __init__(self):
+        hcalnzs.__init__(self)
+        self.recoSeq=':reconstruction_HcalNZS'
+        self.cbSc='pp'
+        self.addEI=True
+        self.isRepacked=True
+        self.eras=Run2_2018_pp_on_AA
+        #keep post-era parts the same as in the default 2018 era
+        self.promptCustoms += [ 'Configuration/DataProcessing/RecoTLR.customisePostEra_Run2_2018' ]
+        self.expressCustoms += [ 'Configuration/DataProcessing/RecoTLR.customisePostEra_Run2_2018' ]
+        self.visCustoms += [ 'Configuration/DataProcessing/RecoTLR.customisePostEra_Run2_2018' ]
```

Comment:
please use the same customs as in ppEra_Run2_2018_pp_on_AA.py
customisePostEra_Run2_2018_pp_on_AA

Reply:
changed to use customisePostEra_Run2_2018_pp_on_AA in the last commit